api/payments/index.ts

```diff
@@ -20,10 +20,11 @@ export async function getPayments(req: EnhancedRequest, res: express.Response) {
     res.status(200)
     res.send(jsonResponse)
   } catch (error) {
-    res.status(error.status).send({
+    next(error)
+    /*res.status(error.status).send({
```

Comment:
can we please remove code rather than commenting it out, there's history for restoring it